hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -23,6 +23,7 @@
 
 #include "hardware_interface/loaned_command_interface.hpp"
 #include "hardware_interface/loaned_state_interface.hpp"
+#include "hardware_interface/types/hardware_interface_status_values.hpp"
```

Comment:
Unused?

Reply:
It is used the type from it:

hardware_interface::hardware_interface_status

Reply:
I see! How come this passed so far without any linters complaining loudly? Or maybe they are already doing that on the other PR in which case maybe this change is more fit there? We won't be merging this into #236 but master, the luggage on that PR is already enormous.

Reply:
Och sorry to do it against 236... after merging it we can rebase it and change PR